From: Lennart Poettering <lennart@poettering.net>
Date: Wed, 10 Feb 2021 10:50:23 +0100
Subject: analyze: slightly reword PrivateTmp= message

Apparently there way confusion about "does not apply". Let's say "is not
appropriate".

Fixes: #13095
(cherry picked from commit 77552b9520ba0d47cbf33cdbe1ddedb9ce9b5bf3)
---
 src/analyze/analyze-security.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/analyze/analyze-security.c b/src/analyze/analyze-security.c
index 8d94fbc..99ec7b5 100644
--- a/src/analyze/analyze-security.c
+++ b/src/analyze/analyze-security.c
@@ -1545,7 +1545,7 @@ static int assess(const struct security_info *info, Table *overview_table, Analy
 
                 if (a->default_dependencies_only && !info->default_dependencies) {
                         badness = UINT64_MAX;
-                        d = strdup("Service runs in special boot phase, option does not apply");
+                        d = strdup("Service runs in special boot phase, option is not appropriate");
                         if (!d)
                                 return log_oom();
                 } else {
